java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/OptionsExample.java

```diff
@@ -0,0 +1,42 @@
+package com.saucelabs.saucebindings.junit5.examples;
+
+import com.saucelabs.saucebindings.SaucePlatform;
+import com.saucelabs.saucebindings.SauceSession;
+import com.saucelabs.saucebindings.junit5.SauceBindingsExtension;
+import com.saucelabs.saucebindings.options.SauceOptions;
+import java.time.Duration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class OptionsExample {
```

Comment:
@titusfortner one of the reasons why we started Sauce Bindings was to increase the amount of code needed to get started. I wonder how much less code this is vs the legacy way?

Reply:
Quite a bit still. I'll add examples without the extension to show it.
But look at quick start example.

Reply:
So, the big difference is the Quick Start; you get all the defaults without any boilerplate:
[with extension](https://github.com/saucelabs/sauce_bindings/blob/c67d25f33578b4151b26295ec96ea83734805524/java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/QuickStartExample.java) (27 lines) vs [without extension](https://github.com/saucelabs/sauce_bindings/blob/c67d25f33578b4151b26295ec96ea83734805524/java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/without/QuickStartExample.java) (93 lines)

But most users are going to have custom capabilities and want to toggle between running on Sauce & running locally. So the "real world" example comparison looks more like this:
[with extension](https://github.com/saucelabs/sauce_bindings/blob/c67d25f33578b4151b26295ec96ea83734805524/java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/ToggleLocalExample.java) (88 lines) vs [without extension](https://github.com/saucelabs/sauce_bindings/blob/c67d25f33578b4151b26295ec96ea83734805524/java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/without/ToggleLocalExample.java) (155 lines)